fs_gcsfs/_version.py

```diff
@@ -1 +1 @@
-__version__ = "0.2.0"
+__version__ = "0.3.0"
```

Comment:
anyway to automate this? (e.g. by using versioneer?)

Reply:
Of course we could but the question is whether we want to add versioneer (that has 2500 LOC and adds 4 or 5 files to this project) to such a small project (<800 LOC) just for avoiding changing this file (which will happen very rarely)